src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -275,10 +275,10 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
             maxLocation = k
             do k = maxLocation, maxLevelCell(iCell)
                BruntVaisalaFreqTopEdge = max(BruntVaisalaFreqTop(k, iCell), 0.0_RKIND)
-
-               RediKappaScaling(k, iCell) = min(max(config_GM_min_stratification_ratio, &
-                                               BruntVaisalaFreqTopEdge/(maxN + 1.0E-10_RKIND)), &
-                                           1.0_RKIND)
+!!!!!!!!! LUKE WILL CHANGE THIS LOGIC, OCT 8 2020
+!               RediKappaScaling(k, iCell) = min(max(config_GM_min_stratification_ratio, &
```

Comment:
@vanroekel please change this line and also 559, where `config_GM_min_stratification_ratio` was removed.